.github/workflows/build_and_test.yml

```diff
@@ -199,10 +255,13 @@ jobs:
     - name: add remote
       run: conan remote add tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs --force
     - name: Build tket
-      run: conan create --profile=tket recipes/tket --build=missing
+      if: needs.check_changes.outputs.tket_changed == 'true'
+      run: conan create --profile=tket recipes/tket tket/stable --build=missing
     - name: Build and run tket tests
+      if: needs.check_changes.outputs.tket_changed == 'true' || needs.check_changes.outputs.tket_tests_changed == 'true'
       run: conan create --profile=tket recipes/tket-tests
     - name: Build and run tket proptests
+      if: needs.check_changes.outputs.tket_changed == 'true' || needs.check_changes.outputs.tket_proptests_changed == 'true'
       run: |
         conan install --profile=tket rapidcheck/cci.20210702@ --build=missing
```

Comment:
What is the flag `rapidcheck/cci.20210702@` doing here?

Reply:
This can probably be omitted if we put `--build=missing` in the following line (but I would do that in a separate PR).

Reply:
Okay, lets keep that in mind!